base-template.sh

```diff
@@ -518,38 +583,88 @@ update_shared_hooks_if_appropriate() {
 
     RUN_UPDATE="false"
     [ "$HOOK_NAME" = "post-merge" ] && RUN_UPDATE="true"
-    [ "$HOOK_NAME" = ".githooks.shared.trigger" ] && RUN_UPDATE="true"
```

Comment:
Maybe we discussed this, but why is this going away?
Was this used from the install script or something? (or maybe cli?)